src/Java.Interop.Tools.Generator/Metadata/FixupXmlDocument.cs

```diff
@@ -168,6 +169,18 @@ public void Apply (ApiXmlDocument apiDocument, string apiLevelString, int produc
 			}
 		}
 
+		public List<NamespaceTransform> GetNamespaceTransforms ()
```

Comment:
Should this return an interface instead of a `List<T>` from a public method in a public class?  Means we can't change the runtime type easily, and -- when taking "duplicates" into consideration -- a `Set<T>` of some form might be ideal?

Reply:
I don't think we consider this public API that we are guaranteeing API compatibility with, but changed anyways.